### ISSUE TITLE
IP Can't connect to my raspberry pi 5

### ISSUE THREAD

**Issue by Fives3Dprint** [LogicReinc/LogicReinc.BlendFarm#100]
I tried connecting the app to my raspberry pi 5 but it simply doesn't work, can u please help me? It asks me a port to use, but none of them are working, can you please help me to choose the right port? Thanks!

**Comment by LogicReinc** [comment]
By default, Blendfarm servers run on port 15000. Also make sure your raspberry allows that port through the firewall, if you use something like UFW firewall, you might have to add ufw allow 15000.

**Comment by Fives3Dprint** [comment]
It still doesn't work, I checked my livebox and my raspberry pi doesn't have one... I don't have the foggiest idea why it doesn't work

**Comment by LogicReinc** [comment]
How are you connecting? Trying to use the auto-discovery, or manually entering the IP+port?
If you're using manual address, add the port. (So if your raspberry ip is "192.168.1.123", enter "192.168.1.123:15000")

**Comment by Fives3Dprint** [comment]
Yeah, that's what I did, I also tried with few different machines but none work... I tried with a CM4 too... It cannot connect
<img width="248" alt="image" src="https://github.com/LogicReinc/LogicReinc.BlendFarm/assets/96694660/cb69eb1d-b78a-400e-8a30-a2b333c73022">
